data/ships.txt

```diff
@@ -95,4 +91,4 @@ ship "Starbus"
 	engine 0 63
 	explode "small explosion" 5
 	explode "small explosion" 10
-	description `A small passenger craft popular in the core systems for ferrying around middle-class people in moderate discomfort. Syndicated Shipyards advertises that travelling in a Starbus hearkens back to the first golden age of air travel on Earth, whatever that might mean.`
+	description `A small passenger craft popular in the Core systems for ferrying around middle-class people in moderate discomfort. Syndicated Shipyards advertises that travelling in a Starbus hearkens back to the first golden age of air travel on Earth.`
```

Comment:
Core: in-game it should be capitalized (ref. `FW Katya 2`’s `on offer` section)
And the player will know what that may mean, and so will `<first>` `<last>`.